_content/lorawan/security.md

```diff
@@ -1,18 +1,32 @@
 ---
 title: Security
+section: Specifications
+zindex: 2
+redirect_from:
+ - current//#security/
```

Comment:
Check this

Reply:
actually I think that the `redirect_from` are not necessary as we mentioned before.

Reply:
Mentioned before where?

Reply:
https://github.com/TheThingsNetwork/docs/pull/319#pullrequestreview-291694400

Reply:
That is exactly what I meant with using `redirect_from`. Those are creating the redirects for SEO.

Reply:
I'm triggered by the double `/` in this URL; that doesn't seem a valid origin